src/commands/syncModel/export/README.md

```diff
@@ -1,38 +1,114 @@
+The `sync-model export` command is an essential part of the content model synchronization process (see the [`sync-model run`](../run/README.md) command). It allows you to generate a local snapshot of the content model from a specified Kontent.ai environment, including Content Types, Content Type Snippets, and Taxonomies. This exported content model is saved in a folder and can be used as the source for synchronization or diff operations.
 
-> [!CAUTION]
-> In case of manual adjustments to the folder's content, make sure to leave the content model in the correct state.
+## Why Export Your Content Model?
+
+Exporting your content model provides several benefits:
+
+- **Stability**: By creating a stable snapshot of your content model, you ensure that the model remains unchanged between diffing and synchronization processes. This prevents discrepancies that can occur if changes are made in the source environment during these operations.
+  
+- **Consistency**: Working with a local copy of the content model guarantees that the diff and sync operations use the exact same data, eliminating the risk of mid-process changes affecting the outcome.
+  
+- **Safety**: It allows you to review and, if necessary, adjust the content model before applying changes to the target environment.
+  
+- **Efficiency**: Reduces the need for constant access to the source environment during synchronization, which can be beneficial in environments with restricted access or when automating deployment processes.
+
+Using the exported content model is considered a safer way of synchronizing content models, especially in environments where multiple team members might be making changes.
+
+## How It Works
+
+The generated model follows the [Management API (MAPI)](https://kontent.ai/learn/docs/apis/openapi/management-api-v2/) format and is processed to remove IDs and other non-essential fields like `last_modified`. References to other entities (e.g., a snippet within a snippet element) are updated to use `codename` instead of `id`. Additionally:
+
+- An `external_id` is set for each entity. If an entity already has an `external_id`, its value is preserved. Otherwise, the value of the entity's `id` is used.
+  
+- Custom properties such as `data-asset-codename`, `data-item-codename`, and `data-codename` are included for convenience when referencing items and assets inside guidelines' rich text. These properties are removed and transformed to the corresponding IDs or external IDs during synchronization.
+
+> **⚠️ Caution**
+>
+> If you manually adjust the content within the exported folder, ensure that the content model remains in a valid state to prevent errors during synchronization.
+
+## Output Files
+
+A successful execution of the command results in the creation of the following files in the specified output folder:
 
- Successful execution of the command results in four files: 
 - `contentTypes.json`
 - `contentTypeSnippets.json`
```

Comment:
well there are many more now :/ we forgot to update it
- `webSpotlight.json`
-  `assetFolders.json`
- `collections.json`
- `spaces.json`
- `languages.json`
- `workflows.json`

Reply:
ok I see, thanks